electron/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import fetch, { Response, RequestInfo, RequestInit } from 'node-fetch';
+import { randomBytes } from 'crypto';
+
+type ReturnType = Response & {
+  headers: { [k: string]: string };
+  url: string;
+  id: string;
+};
+
+class FetchHelper {
+  private static _responses = new Map<string, Response>();
```

Comment:
It is important that `_getResponse` (`getBuffer`, `getText`, `getJson`, `getBlob`) gets called; otherwise there could be a memory leak... The client side library will always do this, but this function is also exported to be used separately...

Reply:
1718da7bcd693d58383ed42c5145f65d9dff3406 should remediate somewhat